cranelift/filetests/filetests/isa/aarch64/call.clif

```diff
@@ -125,8 +115,7 @@ block0(v0: i8):
 ; nextln:  movz x5, #42
 ; nextln:  movz x6, #42
 ; nextln:  movz x7, #42
-; nextln:  sxtb x9, w9
-; nextln:  stur x9, [x8]
+; nextln:  sturb w9, [x8]
```

Comment:
I sort of wish we had a similar ABI test for x64... not a blocker for this PR but it might come in handy to rule things out if we're troubleshooting in the future.

Reply:
For sure, I agree -- I'll try to do this if/when I touch the ABI code next.